tests/functional/client/test_cli_client.py

```diff
@@ -49,79 +49,82 @@ def test_interrupt(monkeypatch, capsys):
         assert trials[0].status == "interrupted"
 
 
-def test_interrupt_diff_code(storage, monkeypatch, capsys):
+def test_interrupt_diff_code(monkeypatch, capsys):
     """Test interruption from within user script with custom int code"""
-    monkeypatch.chdir(os.path.dirname(os.path.abspath(__file__)))
 
```

Comment:
Do you remember why the fixture `storage` would not work correctly anymore when getting rid of the singleton?

Reply:
If the issue is because global config is not set properly, that I think it would be best to fix this inside the fixture so we don't need to add back the OrionState boilerplate everywhere.

Reply:
I don't remember if it was an actual issue.

But I also anted to reduce the number times storage are instantiated, since it made debugging easier to track down.
Some test bugged because they ended up using the wrong storage, so having less instantiation to go through helped a lot.

Reply:
works, done

Reply:
Yeah, that's what I meant. :) Thanks!